### ISSUE TITLE
web client howto refers to `twisted.web._newclient.Response` but should refer to it as `twisted.web.client.Response`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4769]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)| @glyph reported|
|-|-|
|Trac ID|trac#4769|
|Type|enhancement|
|Created|2010-12-12 06:53:49Z|
|Branch|https://github.com/twisted/twisted/tree/web-response-doc-link-4769|

The name is re-exported via an `__all__`.

<details><summary>Searchable metadata</summary>

```
trac-id__4769 4769
type__enhancement enhancement
reporter__glyph glyph
priority__normal normal
milestone__ 
branch__branches_web_response_doc_link_4769 branches/web-response-doc-link-4769
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__web web
keywords__ 
time__1292136829000000 1292136829000000
changetime__1295012404000000 1295012404000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30371]) Branching to 'web-response-doc-link-4769'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30372]) link to twisted.web.client.Response instead of twisted.web._newclient.Response

refs [#4769](https://github.com/twisted/twisted/issues/4769)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [30373]) news fragment

refs [#4769](https://github.com/twisted/twisted/issues/4769)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun removed owner|
|-|-|

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/web-response-doc-link-4769)

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_5"></a>|@cyli set owner to @exarkun|
|-|-|

Looks fine to merge!

**Comment by twisted-trac** [comment]
|[<img alt="cyli's avatar" src="https://avatars.githubusercontent.com/u/223200?s=50" width="50" height="50">](https://github.com/cyli)<a name="note_6"></a>|@cyli commented|
|-|-|

ugh, forgot to delete the review keyword

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_7"></a>|@exarkun set status to `closed`|
|-|-|

(In [30441]) Merge web-response-doc-link-4769

Author: exarkun
Reviewer: cyli
Fixes: [#4769](https://github.com/twisted/twisted/issues/4769)

Change the `Response` link in the HTTP client howto to refer to the public
name of that class.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
